staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -346,13 +346,17 @@ type KubeletConfiguration struct {
 	// Default: "5m"
 	// +optional
 	NodeStatusReportFrequency metav1.Duration `json:"nodeStatusReportFrequency,omitempty"`
-	// nodeLeaseDurationSeconds is the duration the Kubelet will set on its corresponding Lease,
-	// when the NodeLease feature is enabled. This feature provides an indicator of node
-	// health by having the Kubelet create and periodically renew a lease, named after the node,
-	// in the kube-node-lease namespace. If the lease expires, the node can be considered unhealthy.
-	// The lease is currently renewed every 10s, per KEP-0009. In the future, the lease renewal interval
-	// may be set based on the lease duration.
-	// Requires the NodeLease feature gate to be enabled.
+	// nodeLeaseDurationSeconds is a parameter that is relevant when the
+	// NodeLease feature gate is enabled. This feature provides an indicator of
+	// node health by having the Kubelet create and periodically renew a
+	// lease, named after the node, in the kube-node-lease namespace. The
+	// renewal period is one fourth of the `nodeLeaseDurationSeconds`. The
+	// `duration` field of the lease's status is not important. If the
+	// NodeLease feature gate is not enabled then the Kubelet indicates
+	// its node's health by periodically updating status conditions.  The
+	// kube-controller-manager has a parameter, known on the command line as
+	// `--node-monitor-grace-period`, and marks a node unhealthy if the
```

Comment:
As if you should run a few scripts like #99675 said to update the docs.

Reply:
I ran the three scripts cited there.  Surprisingly, they made no change.  Perhaps because this is not the 'droid they're looking for?  See https://gist.github.com/MikeSpreitzer/8fbec06874f7b8a3d3c3ad287c6c3a17
I also tried `make update`, and that also made no change.

Reply:
En. I have the same trouble before. I have fixed the trouble by :  1. delete the "kubernetes" directory ,2. delete the files in the  $GOPATH/bin, 3. checkout the kubernetes from the remote server again, 4. redo the PR with the same branch name. I think that  the cach lead to the trouble. But I am not sure.

Reply:
So I did those steps, and then `make update` made a difference.  Oddly, the difference was not related to my source change!  I then tried `make update FORCE_ALL=true`, and still it made a difference but none related to my source change.  Puzzled here.  See the latest commit, 31e52ad0e79 .

Reply:
Could it be that the in-tree doc generation scripts do _not_ cover code in `staging/src/k8s.io/kubelet` ?